src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -128,12 +146,22 @@ protected void handleGet(RestChannel channel, RestRequest request, Client client
         try {
             builder.startObject();
             final User user = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER);
+            final String username = user.getName();
+            final SecurityDynamicConfiguration<?> internalUser = load(CType.INTERNALUSERS, false);
             if (user != null) {
                 final TransportAddress remoteAddress = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_REMOTE_ADDRESS);
                 final Set<String> securityRoles = privilegesEvaluator.mapRoles(user, remoteAddress);
                 final SecurityDynamicConfiguration<?> configuration = load(getConfigName(), false);
 
-                builder.field("user_name", user.getName())
+               if (request.path().endsWith(SAVED_TENANT)){
+                    if (configuration.exists(user.getName())){
+                        InternalUserV7 iu = (InternalUserV7) internalUser.getCEntry(username);
```

Comment:
Would it be more clear to name the variable using words like `internalUser` instead of abbreviations like `iu`?

Reply:
Also, will be only concerned about InternalUserV7? What about InternalUserV6?

Reply:
> Would it be more clear to name the variable using words like `internalUser` instead of abbreviations like `iu`?

Not really; `internalUser` is already being used elsewhere in the method (see: L230). `iu` is only used one more time in the rest of the function (the following line, L247), so I think it's pretty clear.

Reply:
> Also, will be only concerned about InternalUserV7? What about InternalUserV6?

I was under the impression that we had already migrated/mapped all `InternalUserV6` to `InternalUserV7`; is this not the case?

Reply:
So elasticsearch version 6.8 and below still uses InternalUserV6

Reply:
Support for ES 6.x is currently out of scope.

Reply:
> internalUser is already being used elsewhere in the method (see: L230). 

It doesn't have to be `internalUser`. You can use a different user name, but it will be better to avoid abbreviations as much as possible in variable name.